drizzlepac/drizCR.py

```diff
@@ -377,8 +377,7 @@ def getHelpAsString(docstring=False, show_ver=True):
     if docstring or (not docstring and not os.path.exists(htmlfile)):
         if show_ver:
             helpString = "\n{:s} Version {:s} updated on {:s}\n\n".format(
-                __taskname__, __version__, __version_date__
-            )
+                __taskname__, __version__)
```

Comment:
```python
helpString = f"\n{__taskname__} Version {__version__}\n"
```
There is no date so `"updated on "` is not needed. Moreover, here it would fail because there more format specifications than inputs.